### PR TITLE
added ability to read export_mode from coupling directions block

### PR DESCRIPTION
**Pull request by seb-wahl**
could this be merged soon, it's required for our new people to run NEMO with high resolution nest (NEMO-AGRIF) within ESM-Tools

### CODE REVIEW

File: esm_runscripts/coupler.py [hunk 1]
```diff
@@ -255,20 +255,23 @@ def add_couplings(self, full_config):
                                 if not found_left or not found_right:
                                     sys.exit(0)
 
+                            if "export_mode" in full_config[self.name]:
+                                export_mode = full_config[self.name]["export_mode"]
+                            else:
+                                export_mode = "DEFAULT"
```
Comment (mandresm) [inline esm_runscripts/coupler.py]: Can't this whole block be written as `export_mode = full_config[self.name].get("export_mode", DEFAULT)`?

**Review by mandresm — CHANGES_REQUESTED** [review]

File: esm_runscripts/coupler.py (hunk 1 above)
Comment (seb-wahl) [reply esm_runscripts/coupler.py]: I'm sure it can, since it's a hot fix I better try it first :-)

Comment (seb-wahl) [reply esm_runscripts/coupler.py]: tested and pushed

**Review by mandresm — APPROVED** [review]
I'll merge it now :)

**Pull request merged**